src/uwtools/scheduler.py

```diff
@@ -293,7 +296,9 @@ def pre_process(self) -> Dict[str, Any]:
                 items[RequiredAttribs.TASKS_PER_NODE]
             )
         ] = ""
-        items[RequiredAttribs.NODES] = int(items[RequiredAttribs.TASKS_PER_NODE]) * int(items[RequiredAttribs.NODES])
+        items[RequiredAttribs.NODES] = int(items[RequiredAttribs.TASKS_PER_NODE]) * int(
+            items[RequiredAttribs.NODES]
+        )
```

Comment:
The linter probably did this, but can we give "code readability" a little more weight?